webview_panels/src/uiCore/uiToolkitStories/Components.stories.tsx

```diff
@@ -136,6 +141,20 @@ export const UIKit = {
         >
           hello
         </Drawer>
+
+        <Button onClick={() => setOpenAlert(true)}>Show Alert</Button>
+        <AlertModal
+          isOpen={openAlert}
+          title="Save changes?"
+          actionsFooter={
+            <>
+              <Button>Stay, do not proceed</Button>
+              <Button>Cancel and proceed</Button>
+            </>
+          }
+        >
+          Your changes will be lost!
+        </AlertModal>
```

Comment:
Shouldn't we use vscode api?

Reply:
Using vscode api is better as it would be displayed even if the panel is not selected.

Reply:
that is good idea. it actually works even if switch panels. thank you